hw/ip/hmac/dv/cov/hmac_cov_unr_excl.el

```diff
@@ -0,0 +1,141 @@
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (2) "gen_normal_fifo.full 0,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "3867317506" "gen_normal_fifo.empty" (1) "gen_normal_fifo.empty 0"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 3 "2807735941" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1287846560" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "2429929352" "gen_normal_fifo.fifo_incr_wptr" (0) "gen_normal_fifo.fifo_incr_wptr 1"
+CHECKSUM: "2024123546 4157704780"
+INSTANCE: tb.dut.u_tlul_adapter.u_rspfifo
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (0) "gen_normal_fifo.full 1,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 0 "1862733684" "gen_normal_fifo.full" (2) "gen_normal_fifo.full 0,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "4156807255" "(gen_normal_fifo.fifo_empty && wvalid_i)" (0) "(gen_normal_fifo.fifo_empty && wvalid_i) 1"
+ANNOTATION: "VC_COV_UNR"
+Branch 2 "3867317506" "gen_normal_fifo.empty" (1) "gen_normal_fifo.empty 0"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "2807735941" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (2) "(!rst_ni) 0,0,1,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "1287846560" "(!rst_ni)" (3) "(!rst_ni) 0,0,1,0"
+ANNOTATION: "VC_COV_UNR"
+Branch 6 "2429929352" "gen_normal_fifo.fifo_incr_wptr" (0) "gen_normal_fifo.fifo_incr_wptr 1"
+CHECKSUM: "3042386013 2660487743"
+INSTANCE: tb.dut.u_tlul_adapter
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "1058271942" "(vld_rd_rsp && reqfifo_rdata.error)" (0) "(vld_rd_rsp && reqfifo_rdata.error) 1,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 1 "1058271942" "(vld_rd_rsp && reqfifo_rdata.error)" (1) "(vld_rd_rsp && reqfifo_rdata.error) 0,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 2 "1058271942" "(vld_rd_rsp && reqfifo_rdata.error)" (0) "(vld_rd_rsp && reqfifo_rdata.error) 1,-"
+ANNOTATION: "VC_COV_UNR"
+Branch 2 "1058271942" "(vld_rd_rsp && reqfifo_rdata.error)" (1) "(vld_rd_rsp && reqfifo_rdata.error) 0,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 4 "1984420262" "((reqfifo_rdata.op == OpRead) & (~reqfifo_rdata.error))" (0) "((reqfifo_rdata.op == OpRead) & (~reqfifo_rdata.error)) 1"
+ANNOTATION: "VC_COV_UNR"
+Branch 5 "4197348121" "(!rst_ni)" (1) "(!rst_ni) 0,1"
+ANNOTATION: "VC_COV_UNR"
+Branch 6 "744749108" "reqfifo_rvalid" (1) "reqfifo_rvalid 1,0,1"
```

Comment:
msg fifo is a WO fifo, so no rvalid signal.
Related code: https://github.com/lowRISC/opentitan/blob/master/hw/ip/tlul/rtl/tlul_adapter_sram.sv#L231